### PR TITLE
Fix node selector hostmaps

### LINKED ISSUE #97817: missing nodeSelector for volume tests
similar to #97805 
PR en route

### PR DESCRIPTION
**Pull request by jayunit100**
fixes #97817 
similar to  #97805

### CODE REVIEW

**Comment by jayunit100** [comment]
hold on for #97805 to merge for this one... @jsturtevant @ravisantoshgudimetla

**Comment by jayunit100** [comment]
/test pull-kubernetes-bazel-test
--

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @jayunit100

File: test/e2e/windows/service.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = SIGDescribe("Services", func() {
+var _ = SIGDescribe("[Feature:Windows] Services", func() {
```
Comment (jsturtevant) [inline test/e2e/windows/service.go:32]: I think SIGDescribe already adds [sig-windows] do we need the additional tag for anything?

example:
```
[sig-windows] Services should be able to create a functioning NodePort service for Windows
```

Comment (jayunit100) [reply test/e2e/windows/service.go:32]: hiya , i just added this bc/ @marosset mentioned a similar tag addition in a previous pr :).  figure the redundant info wont hurt.  if any strong feelings happy to remove just lmk

Comment (jayunit100) [reply test/e2e/windows/service.go]: to make this review cleaner i just removed this , and will add that `Feature` tag in a later cleanup, if we decide we want consistency

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

again

**Pull request merged**